src/components/Toggle/Toggle.stories.mdx

```diff
@@ -0,0 +1,180 @@
+import {
+  Meta,
+  Story,
+  Canvas,
+  ArgsTable,
+  Description,
+} from "@storybook/addon-docs/blocks";
+import { withDesign } from "storybook-addon-designs";
+import { HStack, VStack, Grid } from "@chakra-ui/react";
+import Heading from "../Heading/Heading";
+import { getCategory } from "../../utils/componentCategories";
+import DSProvider from "../../theme/provider";
+import Toggle from "./Toggle";
+import { ToggleSizes } from "./ToggleSizes";
+
+<Meta
+  title={getCategory("Toggle")}
+  component={Toggle}
+  decorators={[withDesign]}
+  parameters={{
+    design: {
+      type: "figma",
+      url:
+        "https://www.figma.com/file/qShodlfNCJHb8n03IFyApM/Main?node-id=31581%3A302",
+    },
+    jest: ["Toggle.test.tsx"],
+  }}
+  argTypes={{
+    className: { table: { disable: true } },
+  }}
+/>
+
+# Toggle
+
+| Component Version | DS Version |
+| ----------------- | ---------- |
+| Added             | `0.25.3`   |
+| Latest            | `0.25.3`   |
+
+<Description of={Toggle} />
+
+The Toggle component is used as an alternative for the Checkbox component and returns a boolean response. Like the Checkbox component, the Toggle component can display a label to the right of the toggle and helper/error text below the toggle.
+
+## Best Practices
+
+Toggle components are commonly used for “on/off” switches to perform binary actions that occur immediately after a user “flips the switch.”
+
+The Toggle component label should clarify the action being performed. Labels should be short and to the point, often three words or less.
+
+<Canvas>
+  <Story
+    name="Toggle"
+    args={{
+      labelText: "Test Label",
+      size: "lg",
+      defaultChecked: false,
+      helperText: "This is the helper text!",
+      id: "",
+      invalidText: "",
+      isDisabled: false,
+      isInvalid: false,
+      isRequired: false,
+    }}
+  >
+    {(args) => <Toggle {...args} />}
+  </Story>
+</Canvas>
+
+<ArgsTable story="Toggle" />
+
+## Sizes
+
+<Canvas>
+  <DSProvider>
+    <HStack>
+      <VStack>
+        <Toggle size={ToggleSizes.Small} defaultChecked={true} labelText="On" />
+        <Toggle size={ToggleSizes.Small} labelText="Off" />
+      </VStack>
+      <VStack>
+        <Toggle size={ToggleSizes.Large} defaultChecked={true} labelText="On" />
+        <Toggle size={ToggleSizes.Large} labelText="Off" />
+      </VStack>
+    </HStack>
+  </DSProvider>
+</Canvas>
+
+## Controlled Toggle
```

Comment:
This is great but can you add more details and maybe also the "uncontrolled" version? Similar to what's in the `Select` component https://development-6y1v6na1ceqnmojvc5akrm5ewys3cshs.tugboat.qa/?path=/docs/components-form-elements-select--select#getting-select-data-value or `CheckboxGroup` https://development-6y1v6na1ceqnmojvc5akrm5ewys3cshs.tugboat.qa/?path=/docs/components-form-elements-checkboxgroup--checkbox-group#getting-checkbox-data-values